### PR TITLE
created all components & pages, added routing

### PR DESCRIPTION
**Pull request by aishwarya-bm**
created all components & pages, added routing

NOTE: The commented code is for later use, will uncomment in upcoming branches

preview can be seen here:
https://user-images.githubusercontent.com/66695068/160172776-afae6cfb-e509-4081-a9d9-b466a8f169c6.mp4

### CODE REVIEW

File: src/components/cartprice/CartPrice.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            <span>Coupon code</span>
+            <span>
+              <input value="FLAT50" className="coupon-code text-center" />
+            </span>
+          </div>
+          <div className="coupon-discount d-flex">
+            <span>Coupon discount</span>
+            <span className="green-text">-&#x20b9;600</span>
+          </div> */}
+          <hr></hr>
```
Comment (raman1999) [inline src/components/cartprice/CartPrice.jsx:38]: remove  unnecessary code

File: src/components/categories/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+export default function Categories() {
+  const [categories, setCategories] = useState([]);
+  //   const { dispatch } = useFilterProducts();
+  const getCategories = async () => {
+    try {
+      const { data } = await axios.get("/api/categories");
+      setCategories(() => data.categories);
+    } catch (e) {
+      console.log("error occured", e);
+    }
```
Comment (raman1999) [inline src/components/categories/Categories.jsx:15]: remove console

**Review by rahulcs754 — APPROVED** [review]
The code is clean and good

**Review by raman1999 — APPROVED** [review]
code looks clean, good component breakdown

**Pull request merged**